temba/channels/models.py

```diff
@@ -2013,10 +2013,8 @@ def send_vumi_message(cls, channel, msg, text):
         is_ussd = channel.channel_type in Channel.USSD_CHANNELS
         channel.config['transport_name'] = 'ussd_transport' if is_ussd else 'mtech_ng_smpp_transport'
 
-        if msg.response_to_id:
-            in_reply_to = Msg.objects.get(pk=msg.response_to_id).external_id
-        else:
-            in_reply_to = None
+        in_reply_to = Msg.objects.values_list(
+            'external_id', flat=True).get(pk=msg.response_to_id)
```

Comment:
Also was somewhat dubious of what kind of SQL this would turn into, but just tested it and it is indeed right. Django is weird that way.

```python
Msg.objects.values_list('external_id', flat=True).get(pk=42)
```

```sql
SELECT "msgs_msg"."external_id" FROM "msgs_msg" WHERE "msgs_msg"."id" = 42
```